src/lcm/model_functions.py

```diff
@@ -74,6 +188,26 @@ def _big_u(utility, continuation_value, params):
     )
 
 
+def get_multiply_weights(stochastic_variables):
+    """Get multiply_weights function.
+
+    Args:
+        stochastic_variables (list): TODO
+
+    Returns:
+        callable
+
+    """
+    arg_names = [f"weight_next_{var}" for var in stochastic_variables]
+
+    @allow_kwargs
```

Comment:
Need a comment that `allow_kwargs` is here just a lazy way of not consolidating args and kwargs inside `_outer`

Reply:
The clean way would to do this would be to write a `all_as_args` as analogue to the `all_as_kwargs` function. 

We need `lcm.functools`!